crates/fraud_proofs/global_merkle_root/storage/Cargo.toml

```diff
@@ -22,6 +22,10 @@ num_enum = { workspace = true }
 strum = { workspace = true }
 strum_macros = { workspace = true }
 
+[dev-dependencies]
+fuel-core-storage = { workspace = true, features = ["alloc", "test-helpers"] }
```

Comment:
i think std is enabled by default for `fuel-core-storage`, might want to use no-default-features here?